posthog/api/query.py

```diff
@@ -257,6 +251,34 @@ def upgrade(self, request: Request, *args, **kwargs) -> Response:
         upgraded_query = upgrade(request.data)
         return Response({"query": upgraded_query["query"]}, status=200)
 
+    @extend_schema(
+        description="Get query log details from query_log_archive table for a specific query_id",
+        responses={200: "Query log details"},
+    )
+    @action(methods=["GET"], detail=True, url_path="log")
+    def get_query_log(self, request: Request, pk: str, *args, **kwargs) -> Response:
+        try:
+            query = HogQLQuery(
+                query="select * from query_log where query_id = '{client_query_id}'",
```

Comment:
You shouldn't need the single quotes around the placeholder - it'll insert them for you